lib/kovid/historians.rb

```diff
@@ -88,12 +89,16 @@ def histogram(country, date_string)
         dates.each_with_index do |val, index|
           data << [val, positive_cases_figures[index]]
         end
-        y_range = AsciiCharts::Cartesian.new(data, bar: true, hide_zero: true).y_range
+        y_range = AsciiCharts::Cartesian.new(
+          data, bar: true, hide_zero: true
+        ).y_range
 
         last_two_y = y_range.last 2
         y_interval = last_two_y.last - last_two_y.first
 
-        scale("Scale on Y: #{y_interval}:#{(y_interval / last_two_y.last.to_f * positive_cases_figures.last).round(2) / y_interval}")
+        scale("Scale on Y: #{y_interval}:#{(
```

Comment:
💪

Reply:
I tried to move some parts of this calculation to other variables, but did not find any suitable names for them, so ended up just breaking it in multiple lines. xD :eyes:

Reply:
That's fine.